msm/skill_entry.py

```diff
@@ -217,9 +218,8 @@ def run_pip(self, constraints=None):
 
         LOG.info('Installing requirements.txt for ' + self.name)
         can_pip = os.access(dirname(sys.executable), os.W_OK | os.X_OK)
-        pip_args = [
-            sys.executable, '-m', 'pip', 'install'
-        ] + self.dependent_python_packages
+        pip_args = [sys.executable, '-m', 'pip', 'install']
+        pip_args += [self.dependent_python_packages]
```

Comment:
I think this causes an issue. `self.dependent_python_packages` is a list so this adds the entire list as a single argument to Popen.

Reply:
Wow, I'm stupid! Nice catch!